es6.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+module.exports = {
+  extends: ['eslint-config-mm/es5'],
+  parserOptions: {
+    ecmaVersion: 6
+  // Can't use ES6 imports yet in node: https://github.com/nodejs/help/issues/53#issuecomment-165763446
+  // sourceType: 'module'
+  },
+  env: {
+    es6: true
+  },
+  rules: {
+    'arrow-parens': [2, 'always'],
+    'arrow-body-style': [2, 'always'],
+    'arrow-spacing': 2,
+    'no-const-assign': 2,
+    'no-confusing-arrow': 2,
+    'no-var': 1,
+    'object-shorthand': 1,
```

Comment:
I personally prefer defining object properties explicitly instead of this shorthand function. To me it feels messy to leave the object notation with a property and a value and throw in functions and variables directly into an object. It can also lead to lazy behavior of a dev by using a internal variable name as property name in an object, without thinking whether that property name is the best representation of the value within the object.

Purely hypothetical example:
```
const myCustomConfig = require('config/myconfig.json');

// myCustomConfig is the shorthand here, using the internal variable name
module.exports = {
  myCustomConfig
};

// while config might be a better property for this object
module.exports = {
  config: myCustomConfig
};
```

Of course, this seems like a dumb example, but I think not having this rule enforces you to think about the property names you use.